src/climate_learn/models/hub/persistence.py

```diff
@@ -0,0 +1,13 @@
+# Third party
+from torch import nn
+
+
+class Persistence(nn.Module):
```

Comment:
I think the classe should be decorated using register (register("persistence")).